src/live-programming.rst

```diff
@@ -16,6 +16,379 @@ REPLs
 
 Data Analysis Environments
 ==========================
+Data scientists use data analysis environments for a variety of data science activities.
+With the notion of extracting knowledge and insights from data, the term data science was first distinguished from pure statistics in 1997 :cite:`donoho:2017`. 
```

Comment:
Start this sentence with "The term data science was first distinguished from pure statistics in 1997"